pr_stats.c

```diff
@@ -2528,7 +2528,7 @@ __print_funct_t stub_print_filesystem_stats(struct activity *a, int curr, int di
 		       sfc->f_files - sfc->f_ffree,
 		       sfc->f_files ? SP_VALUE(sfc->f_ffree, sfc->f_files, sfc->f_files)
 				    : 0.0,
-		       sfc->fs_name);
+		       DISPLAY_MOUNT(a->opt_flags) ? sfc->mountp : sfc->fs_name);
```

Comment:
Thanks, have sent pull requests for these now.  Broke them down as below.
json https://github.com/sysstat/sysstat/pull/53
xml https://github.com/sysstat/sysstat/pull/54
rndr https://github.com/sysstat/sysstat/pull/55